metadata_gui_panel.py

```
@@ -184,15 +237,18 @@ def __init__(self, comic_data: GUIComicInfo) -> None:
         title_cover_box.addWidget(thumbnail_label)
-        title_cover_box.addWidget(QLabel(metadata.name))
 
         stars = self.make_star_rating(rating=metadata.rating if metadata.rating else 0)
 
-        content_layout.addWidget(title_cover_widget, 0, 0, 1, 1)
-        content_layout.addWidget(creators_box, 1, 0, -1, 1)
-        content_layout.addWidget(description_box, 0, 1, 1, 2)
-        content_layout.addWidget(stars, 0, 3, 1, 1)
-        content_layout.addWidget(review_area, 1, 1, 1, 2)
+        title_cover_box.addWidget(self.create_overview_widget(metadata, stars))
+        overview_panel = DashboardBox("Overview", False)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)